README.md

```diff
@@ -34,3 +34,9 @@ libbpf-cargo = "0.12"
 ```
 
 See [full documentation here](https://docs.rs/libbpf-cargo).
+
+## Contributing
+
+We welcome all contributions! Please see the [contributor's
+guide](https://github.com/libbpf/libbpf-rs/blob/master/CONTRIBUTING.md) for more
```

Comment:
Could we just use a relative path instead of an absolute URL here? That would make it easier to navigate the document locally and in non-GitHub environments.

Reply:
Nice, I didn't realize Markdown supports relative links! https://stackoverflow.com/questions/7653483/github-relative-link-in-markdown-file Will make this change!